api/src/main/java/com/velocitypowered/api/proxy/ProxyServer.java

```diff
@@ -45,23 +46,24 @@
     int getPlayerCount();
 
     /**
-     * Retrieves a registered {@link ServerInfo} instance by its name. The search is case-insensitive.
+     * Retrieves a registered {@link RegisteredServer} instance by its name. The search is case-insensitive.
      * @param name the name of the server
      * @return the registered server, which may be empty
      */
-    Optional<ServerInfo> getServerInfo(String name);
+    Optional<RegisteredServer> getServerInfo(String name);
```

Comment:
`getServerInfo` -> `getServer`/`getRegisteredServer`

Reply:
We don't need `getServer()` returning `ServerInfo`, considering you can do `proxy.getServer("test").map(RegisteredServer::getServerInfo)`.

I will rename `getServerInfo` to `getServer` though.